tailwind.config.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
   mode: "jit",
-  purge: [
-    "./app/pages/**/*.{js,ts,jsx,tsx}",
-    "./app/components/**/*.{js,ts,jsx,tsx}",
-  ],
+  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
```

Comment:
Technically, but this is going anyway
```suggestion
  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./app/**/*.{js,ts,jsx,tsx}"],
```